porch/pkg/oci/mutate.go

```diff
@@ -51,21 +51,40 @@ func (r *ociRepository) CreatePackageRevision(ctx context.Context, obj *api.Pack
 		return nil, err
 	}
 
+	if obj.Spec.Description == "" {
```

Comment:
This seems like the same checks as we do for git?

Reply:
Yeah, it's the same, are we expecting different checks?